AppCenterData/AppCenterDataTests/MSDataTests.m

```diff
@@ -1937,13 +1939,17 @@ - (void)testReadsFromRemoteIfNotExpiredAndOnlineWithNoPendingOperation {
   // Mock document in local storage.
   id<MSDocumentStore> localStorageMock = OCMProtocolMock(@protocol(MSDocumentStore));
   self.sut.dataOperationProxy.documentStore = localStorageMock;
-  NSData *jsonFixture = [self jsonFixture:@"validTestDocument"];
+  NSData *jsonFixture = [self jsonFixture:@"validTestUserDocument"];
   MSDocumentWrapper *expectedDocumentWrapper = [MSDocumentUtils documentWrapperFromData:jsonFixture
                                                                            documentType:[MSDictionaryDocument class]
+                                                                              partition:@"user-123"
```

Comment:
You don't need a custom prefixed user partition it's only used for the user table in the storage. These calls are more likely to use `kMSDataUserDocumentsPartition`.

Reply:
I was wrong the returned metadata from the document wrapper should contain the user-account.